server/src/main/java/keywhiz/utility/DSLContexts.java

```diff
@@ -49,6 +49,6 @@ public static DSLContext databaseAgnostic(DataSource dataSource) throws SQLExcep
     return DSL.using(dataSource, dialect,
             new Settings()
                 .withRenderSchema(false)
-                .withRenderNameStyle(RenderNameStyle.AS_IS));
+                .withRenderQuotedNames(RenderQuotedNames.EXPLICIT_DEFAULT_QUOTED));
```

Comment:
According to the documentation, these flags are independent of each other:

https://www.jooq.org/doc/latest/manual/sql-building/dsl-context/custom-settings/settings-name-style/

Should we be adding a new line rather than replacing the existing one? I guess it's not clear to me what problem un-setting the existing setting is solving.

Reply:
Reading a bit further, AS_IS and EXPLICIT_DEFAULT_QUOTED are both (supposed to be) the default values.